qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/FindPatientPage.java

```diff
@@ -26,7 +26,7 @@ public void enterPatient(String patient) {
 		setTextToFieldNoEnter(PATIENT_SEARCH, patient);
 	}
 
-	public ClinicianFacingPatientDashboardPage clickOnFirstPatient() {
+	public ClinicianFacingPatientDashboardPage clickOnReturnedPatient() {
```

Comment:
Why renaming this method?

Reply:
Its also have the same functionality as those we changed.

Reply:
To bring out the aspect of readability in response to what gets executed on to the server.

Reply:
Does the suggested method stand when the search returns more than one patient?

Reply:
The search must only return one patient. that's why we are changing the naming. so the suggested method stands when search returns one patient.

Reply:
Isn't there possibility when the search indexing criteria returns more than one patient?

Reply:
No possibility because the system generates a single particular identifier per patient.

Reply:
In the case when searching by patient name,  the system may return more than one patient and the earlier method name makes meaning.

Reply:
Still, at the end of the day, we expect to click on the returned patients.  we dont want to assume there are morethan one patient using either name or identifier, there should be uniquness.

Reply:
It's possible to have active patients with similar name and in that case when searching using patient name the search indexing criteria will return all of the patients so the current method name makes meaning compared to the suggested one.

Reply:
Like forexample. Do you mind giving me that name such that i can test it out.

Reply:
Search the patient by name "Risty" on the https://demo.openmrs.org/openmrs/coreapps/activeVisits.page?app=coreapps.activeVisits and see the return.

Reply:
How about when you use patient identifier, does it still return the two names.

Reply:
Searching by identifier returns one patient because each patient has a unique identifier.  In the OpenMRS world patient search is done by name since system users may not be in position to know each patient identifier.

Reply:
> Searching by identifier returns one patient because each patient has a unique identifier.

This is what we want to achieve than returning many patients, And forexample  how many patients gets returned when you use Risty Allen.

Reply:
Otherwise @dkayiwa  What is your take on this part and according to @kdaud  comment.

Reply:
The click on first patient that i commented on, was searching using patient identifier. If it was using patient name, then i would not complain about it.

Reply:
Ok then. thanks.

Reply:
If you use clickOnFirstPatient when searching by name, then i will not complain. But if you use the same clickOnFirstPatient when searching by identifier, then i will shout at you again. 😊

Reply:
ohh, then my recommendation is , either we maintain clickOnFirstPatient as a method name from findPatientPage, or `clickOnReturnedPatient(). ` but for the case of gherkin , we should use first patient for searching by name and clickOn returned patient when searching by identifier, Question, Are we mainting this method naming or i revert it back to default.

Reply:
The current method name(`clickOnFirstPatient()`) is good enough to do the work, one thing I would go for is changing the gherkin syntax where applicable.

Reply:
Thanks. Pushed that in the last commit.